bin/deploy_automation/automated_release_cycle.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -euo pipefail
+SCRIPT_DIR=$(dirname "$0")
+source "$SCRIPT_DIR/_global_vars.sh"
+source "$SCRIPT_DIR/_shared_functions.sh"
+
+# Execute release cycle basic automation
```

Comment:
did you have your definition of "release cycle" written somewhere?

Reply:
Not really. It is embedded to these shell descriptions, but I'll add it to the readme file as suggested in our meeting today.